### PR TITLE
Enable Proxy debugging

### PR DESCRIPTION
**Pull request by iamgusain**
Enabling proxy debugging to be able to capture network traffic from the test app via fiddler
Followed the instructions from: https://docs.telerik.com/fiddler-everywhere/get-started/mobile-traffic/configure-android#inspect-android-application-traffic

### CODE REVIEW

File: testapps/testapp/src/main/res/xml/network_security_config.xml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<network-security-config>
+    <debug-overrides>
+        <trust-anchors>
+            <!-- Trust preinstalled CAs -->
+            <certificates src="system" />
+            <!-- Trust user added CAs while debuggable only -->
+            <certificates src="user" />
+        </trust-anchors>
+    </debug-overrides>
+</network-security-config>
```
Comment (shahzaibj) [inline testapps/testapp/src/main/res/xml/network_security_config.xml:10]: add new line at end of file

**Review by shahzaibj — APPROVED** [review]

File: testapps/testapp/src/main/res/xml/network_security_config.xml (hunk 1 above)
Comment (iamgusain) [reply testapps/testapp/src/main/res/xml/network_security_config.xml:10]: done

**Review by gitaumoses4 — APPROVED** [review]

File: testapps/testapp/src/main/AndroidManifest.xml [hunk 2]
```diff
@@ -34,6 +34,7 @@
     <uses-permission android:name="android.permission.USE_CREDENTIALS" />
 
     <application
+        android:networkSecurityConfig="@xml/network_security_config"
         android:name="com.microsoft.identity.client.testapp.MsalSampleApp"
         android:allowBackup="true"
```
Comment (iambmelt) [inline testapps/testapp/src/main/AndroidManifest.xml:39]:
```suggestion
        android:allowBackup="false"
```

Comment (iambmelt) [reply testapps/testapp/src/main/AndroidManifest.xml:39]: This is unrelated to your changes, but there's no reason to set this value to true in our testapp 😄

**Review by iambmelt — APPROVED** [review]

**Comment by iamgusain** [comment]
The tests are failing due to a known issue (new http client shadow) in the updated common submodule

**Pull request merged**